src/app/views/index.html

```diff
@@ -7,7 +7,8 @@
 
 		<div class="search">
 			<i class="material-icons">search</i> 
-			<input type=search class="input-search" ng-model="i_search" placeholder="Type the name of your favorite artist" ng-change="alert('holi')">
+			<input type=search class="input-search" ng-model="i_search" placeholder="Type the name of your favorite artist" ng-change="mainCtrl.changeView('/results')">
+			<a href="/results">Holi</a>
 		</div>
 	</div>
 	<div class="favorites-container" ng-show="mainCtrl.has_favorites()">
```

Comment:
you should assign controller to template in routing file insted of doing it here